src/metax_api/tests/api/rest/base/views/datasets/write.py

```diff
@@ -195,7 +195,7 @@ def _set_cr_owner_and_state(self, cr_id, state, owner):
         cr.data_catalog_id = DataCatalog.objects.get(catalog_json__identifier=END_USER_ALLOWED_DATA_CATALOGS[0]).id
         cr.force_save()
 
-    def test_field_exists(self):
+    def test_field_state_exists(self):
```

Comment:
This is not directly related, but I realized the old test name is not telling too much, especially now when we are testing that another field ('issued') exists